x-pack/plugins/aiops/server/lib/is_request_aborted_error.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isPopulatedObject } from '@kbn/ml-is-populated-object';
+
+export function isRequestAbortedError(e: unknown): boolean {
```

Comment:
Can `e` be changed to type `Error`?
I see that it's called in `reportError` where `error` is `unknown`, but `reportError` is only ever passed things of `any` type, so `unknown` could be changed to `any` there.

Reply:
I pushed a new version that's now a real type guard like the others we have (`(arg: unknown): arg is ...`) in eb2a23f.

The reason I used `unknown` in the `reportError` is because we pass on the `reason` from a rejected promise via `allSettled()` and TS treats that as `any` as far as I can see.